requirements.txt

```diff
@@ -14,3 +14,4 @@ Flask-Script==2.0.5
 workdays==1.4
 unicodecsv==0.14.1
 cryptography==1.6
+git+https://github.com/alphagov/notifications-python-client.git@4.0.0#egg=notifications-python-client==4.0.0
```

Comment:
Only official support. It works fine on py2. However I think what it also means we can't _assume_ py2 support going forward.
I can't see any point in implementing backwards incompatibility unless they absolutely have to but it means we need to be careful until the officially support py2.